contracts/token/ERC721/IERC721.sol

```diff
@@ -42,12 +41,37 @@ interface IERC721 is IERC165 {
      * either {approve} or {setApprovalForAll}.
      */
     function transferFrom(address from, address to, uint256 tokenId) external;
+
+    /**
+     * @dev Approves the transfer of a specific NFT (`tokenId`) to an account (`to`)
+     *
+     * Requirements:
+     * - The caller must own the token or be an approved operator
+     */
     function approve(address to, uint256 tokenId) external;
+
+    /**
+     * @dev Gets the approved account for a specific NFT (`tokenId`)
```

Comment:
```suggestion
     * @dev Returns the account approved for transferring the `tokenId` token.
     * 
     * Note that this only returns accounts approved for a single token via 
     * {approve}. For account operators approved via {setApprovalForAll}, use 
     * {isApprovedForAll} instead.
     *
     * Requirements:
     * - `tokenId` must exist.
```

I included a few general guidelines here:
 - prefer 'returns' to 'gets'
 - avoid referring to NTFs and talk about 'tokens' instead (it looks like some of the original docs used NFTs, we'd have to change that)
 - try to integrate the argument names into the sentence, if possible

The note about setApprovalForAll and isApprovedForAll is more of a nice to have :stuck_out_tongue: It requires more in-depth knowledge about the standard.